src/quicktype-core/language/CPlusPlus.ts

```diff
@@ -959,8 +995,37 @@ export class CPlusPlusRenderer extends ConvenienceRenderer {
         );
     }
 
+    protected emitGenerators(): void {
+
+        this.emitNamespaces(this._namespaceNames, () => {
+            this.forEachTopLevel(
+                "leading",
+                (t: Type, name: Name) => this.emitTopLevelTypedef(t, name),
+                t => this.namedTypeToNameForTopLevel(t) === undefined
+            );
+            this.ensureBlankLine();
+        });
+
+        if (!this._options.justTypes && this.haveNamedTypes) {
+            this.emitNamespaces(["nlohmann"], () => {
+                this.forEachObject("leading-and-interposing", (c: ClassType, className: Name) =>
```

Comment:
Just for future reference: TypeScript will infer the argument types for these anonymous functions.